src/target.zig

```diff
@@ -143,6 +158,8 @@ pub fn libcNeedsLibUnwind(target: std.Target) bool {
         .windows => target.abi != .msvc,
         else => true,
     };
+
+    return has_libunwind and needs_libunwind;
```

Comment:
`needs_libunwind and !has_libunwind` is an error condition that should be bubbled up to the caller.